plugins/in_kubernetes_events/kubernetes_events_conf.c

```diff
@@ -218,7 +221,7 @@ struct k8s_events *k8s_events_conf_create(struct flb_input_instance *ins)
     flb_plg_info(ctx->ins, "API server: %s", ctx->kube_url);
 
```

Comment:
I think something like this is necessary:

https://github.com/fluent/fluent-bit/blob/master/src/flb_input.c#L2126C5-L2126C27